update_deps/build_graph.ps1

```diff
@@ -24,7 +24,7 @@ Prints order in which repositories must be updated to file order.json
 
 .EXAMPLE
 
-PS> .\build_graph.ps1 https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-MessagingStore
+PS> .\build_graph.ps1 -root_list 'https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-Messaging-GeoReplication' 'https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-Messaging-ElasticLog'
```

Comment:
>aph.ps1 -root_list 'https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-Messaging-GeoReplication' 'https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-Messaging-ElasticLog' [](http://example.com/codeflow?start=14&length=200)

does it need to be comma-separated as mentioned above and elsewhere or space separated as in this example?

Reply:
Fixed the example